docs/docs/configuration/generated-source.mdx

```diff
@@ -63,3 +63,8 @@ You won't see the updated mapper code or mapper diagnostics until you perform a
 This is done for performance reasons,
 otherwise the IDE could become laggy.
 :::
+
+## Excluding from code coverage
+
+Mapperly decorates each generated mapper type with the `GeneratedCodeAttribute`. This attribute can be used
```

Comment:
It is probably also worth noting the `.g.cs` and the `/// <auto-generated />` comment.

Reply:
I will add that and make it a bit more generic. It could indeed be used for any static code/security analysis tooling.

Reply:
Updated the documentation as per your suggestions, please check if it is satisfactory.